Cargo.toml

```
@@ -31,6 +31,7 @@ zeroize = { version = "1.5", features = ["alloc"] }
 serde = { version = "1.0.103", optional = true, default-features = false, features = ["derive"] }
 sha1 = { version = "0.10.5", optional = true, default-features = false, features = ["oid"] }
 sha2 = { version = "0.10.6", optional = true, default-features = false, features = ["oid"] }
+spki = { version = "0.7.2", optional = true, default-features = false }
```

`rsa` has a hard dependency on `alloc` so you might as well just...

```suggestion
spki = { version = "0.7.2", optional = true, default-features = false, features = ["alloc"] }
```